patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1359,12 +1359,21 @@
 +        return false;
 +    }
 +
++    /**
++     * Set the PathNodeType for this block. null for vanilla behavior.
++     * @return the PathNodeType
++     */
++    @Nullable public net.minecraft.pathfinding.PathNodeType getAiPathNodeType(IBlockState state, IBlockAccess world, BlockPos pos)
```

Comment:
Yeah, but it doesn't really make sense here, the `Block` patch is a huge blob all all already anyways. I just forgot to fix this in my update.